src/Foundation/NSFileManager.cs

```diff
@@ -425,6 +426,59 @@ internal static NSFileSystemAttributes FromDictionary (NSDictionary dict)
 	}		
 	
 	public partial class NSFileManager {
+
+		[DllImport (Constants.FoundationLibrary)]
+		static extern IntPtr NSUserName ();
+
+		public static string UserName {
```

Comment:
A bit confused, or maybe i'm looking at the wrong place. Checking the header of `NSFileManager.h` - doesn't have `NSUserName`. Could only find `FOUNDATION_EXPORT NSString *NSUserName(void);` in `NSPathUtilities.h`. How do we know its a static property of the class?

Reply:
It is not a static property in the class. Is a C function. But I need to place it somewhere and NSFileManager is a class that makes sense. 

The C function does not need the object, so we don't care. But we need and object to expose the function.